hera_cal/redcal.py

```diff
@@ -130,14 +131,16 @@ def get_reds(antpos, pols=['nn'], pol_mode='1pol', bl_error_tol=1.0):
 
 
 def filter_reds(reds, bls=None, ex_bls=None, ants=None, ex_ants=None, ubls=None, ex_ubls=None, 
-                pols=None, ex_pols=None, antpos=None, min_bl_cut=None, max_bl_cut=None):
+                pols=None, ex_pols=None, antpos=None, min_bl_cut=None, max_bl_cut=None, max_dims=None):
     '''
     Filter redundancies to include/exclude the specified bls, antennas, unique bl groups and polarizations.
-    Arguments are evaluated, in order of increasing precedence: (pols, ex_pols, ubls, ex_ubls, bls, ex_bls,
-    ants, ex_ants).
+    Also allows filtering reds by removing antennas so that the number of tip/tilt degeneracies is no more
```

Comment:
I think the fact that arbitrary additional numbers of redcal degeneracies are tip/tilt is somewhat confusing. Can we interpret them as a physical tip/tilt or, tip-tilts of different sub-arrays, are they even more general? If they aren't simply tip/tilt, I think you should phrase it as "additional degeneracies beyond tip/tilt" and maybe add more explanation if they are, for example, tip/tilts of sub-arrays or something similar. If they are tip/tilt explain what it means to describe tip/tilt with more than two degenerate parameters.